packages/cli/generators/project/templates/.vscode/launch.json.ejs

```diff
@@ -0,0 +1,38 @@
+{
+  // Use IntelliSense to learn about possible Node.js debug attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Program",
+      "program": "${workspaceFolder}/index.js"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Run Mocha tests",
+<% if (project.loopbackBuild) { -%>
+      "autoAttachChildProcesses": true,
+      "program": "${workspaceRoot}/node_modules/.bin/lb-mocha",
```

Comment:
IIRC, you need to invoke `_mocha` directly if you want to debug the tests.

`lb-mocha` creates a child process that creates another node process `mocha` which creates the third child process `_mocha.

Maybe we should add a new option to `lb-mocha` to tell our wrapper to start mocha in a debug mode? I am not sure if that would work with VS Code inspector though.

Reply:
If it's easier, then I am ok with leaving out `launch.json` configuration from this initial pull request.

Reply:
We can already start mocha in debug mode by passing in `--inspect` as an argument, but that doesn't seem to fix the problem of the timeout issue when restarting. This might be fixable if lb-mocha calls `_mocha` instead of `mocha`, but I think I'd rather address this in a separate issue.

Reply:
Fair enough 👍  Looks like you already understand this matter better than I do 😄